streampark-common/src/main/java/org/apache/streampark/common/enums/ApplicationType.java

```diff
@@ -32,7 +31,10 @@ public enum ApplicationType {
   STREAMPARK_SPARK(3, "StreamPark Spark"),
 
   /** Apache Spark */
-  APACHE_SPARK(4, "Apache Spark");
+  APACHE_SPARK(4, "Apache Spark"),
+
+  /** Unknown type replace null */
+  UNKNOWN(5, "unknown");
```

Comment:
I agree to add a new type "UNKNOWN", but I have a suggestion regarding the value. I recommend setting the value as -1. Currently, the value is 5. If we add another engine type in the future, it may cause trouble.

Reply:
+1